### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by asmega**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,27 @@
-# Form Builder Publisher
 [![CircleCI](https://circleci.com/gh/ministryofjustice/fb-publisher/tree/master.svg?style=svg)](https://circleci.com/gh/ministryofjustice/fb-publisher/tree/master)
 
+# Form Builder Publisher
+
+This Ruby on Rails app allows users to deploy their forms in to [https://github.com/ministryofjustice/cloud-platform-environments](Cloud Platform)
+
+## Environments
+
+The app provides 2 environments to users. A `development` environment to test and preview changes and a `production` environment for public release
```
Comment (sequencemedialimited) [inline README.md]: Says `development` and `production` ...

File: README.md [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+
+The Live environment can be found at https://fb-publisher-live.apps.live-1.cloud-platform.service.justice.gov.uk/services
```
Comment (sequencemedialimited) [inline README.md]: ... Then immediately says `live` ... is that production?

File: README.md (hunk 1 above)
Comment (brenetic) [inline README.md]: Just a suggestion: A `development` (test) environment to preview changes and a `production` (live) environment for public release.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
+
+Developers have a clone of this setup, the `test` environment which can be found at https://fb-publisher-test.apps.live-1.cloud-platform.service.justice.gov.uk/services
```
Comment (sequencemedialimited) [inline README.md]: ... And there's a `test` so that's three?

Comment (brenetic) [reply README.md]: 🙈

**Pull request merged**